src/modules/user/user.controller.ts

```diff
@@ -0,0 +1,41 @@
+// user.controller.ts
+import {
+  Post,
+  Controller,
+  Put,
+  Param,
+  Body,
+  HttpException,
+  ForbiddenException,
+} from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+
+  @Post()
+  async login(
+    @Body('email') email: string,
```

Comment:
Use class-validator, does this validate the request? Please use this [class-validator](https://github.com/typestack/class-validator).